src/natcap/invest/coastal_blue_carbon/coastal_blue_carbon.py

```diff
@@ -295,49 +307,59 @@
+                        "the LULC classes that it transitions to. Each cell "
+                        "in the matrix is filled with an option indicating "
+                        "the effect on carbon when transitioning from that "
+                        "cell's row's LULC class to that cell's column's LULC "
+                        "class. The classes represented in this table must "
+                        "exactly match the classes in the biophysical table "
+                        "``lulc-class`` column. A cell may be left empty if "
+                        "the transition never occurs.")}
             },
             "about": (
                 "A transition matrix mapping the type of carbon action "
-                "undergone when one landcover type transitions to another. "
-                "The Coastal Blue Carbon preprocessor exists to help create "
-                "this table for you."),
+                "undergone when one landcover type transitions to another."),
         },
         "do_economic_analysis": {
-            "name": "Calculate Net Present Value of Sequestered Carbon",
+            "name": "Do Valuation",
             "type": "boolean",
             "required": False,
             "about": (
-                "A boolean value indicating whether the model should run an "
-                "economic analysis."),
+                "Enable net present valuation analysis based on "
+                "carbon prices from either a yearly price table, or an "
+                "initial price and yearly interest rate."),
         },
         "use_price_table": {
             "name": "Use Price Table",
             "type": "boolean",
             "required": False,
             "about": (
-                "boolean value indicating whether a price table is included "
-                "in the arguments and to be used or a price and interest rate "
-                "is provided and to be used instead."),
+                "Use a yearly price table, rather than an initial "
+                "price and interest rate, to indicate carbon value over time."),
         },
         "price": {
             "name": "Price",
             "type": "number",
             "units": u.currency/u.megatonne,
             "required": "do_economic_analysis and (not use_price_table)",
-            "about": "The price of CO2E at the base year.",
+            "about": (
+                "The price of CO2E at the baseline year. Required if Do "
+                "Valuation is selected and Use Price Table is not selected."),
         },
         "inflation_rate": {
             "name": "Interest Rate",
             "type": "percent",
             "required": "do_economic_analysis and (not use_price_table)",
-            "about": "Annual change in the price per unit of carbon.",
+            "about": (
+                "Annual increase in the price of CO2E. Required if Do "
+                "Valuation is selected and Use Price Table is not selected.")
```

Comment:
I can see how this would be more annoying to edit in the future. For me this was less about standardizing phrasing and more about:
- reducing duplicated text to reduce translation time/costs
- ensuring we document the "condition" for conditionally-required args. most conditionally-required args simply depend on one other arg and we could auto-generate that description. but we allow the condition to be any python expression, and a few args use that feature. so auto-documenting is not really possible. I was hoping to get close to that but still allow a custom description for more complicated conditions.

There may not be a perfect solution here, I'm pretty ambivalent about it.

Reply:
> reducing duplicated text to reduce translation time/costs

I often forget about this part of it. It will take a little while to get in the mindset of being aware of the translation impact when working on future user guide changes.

Reply:
> > reducing duplicated text to reduce translation time/costs
> 
> I often forget about this part of it. It will take a little while to get in the mindset of being aware of the translation impact when working on future user guide changes.

It's true, but I wouldn't want us to go too far out of our way to conserve words. If we can afford the translation costs in general, then I'm sure we can afford a few extra words here and there.